github/event_types.go

```diff
@@ -294,6 +294,26 @@ type PullRequestEvent struct {
 	Sender  *User       `json:"sender,omitempty"`
 }
 
+// PullRequestReviewEvent is triggered when a review is submitted on a pull
+// request.
+// The Webhook event name is "pull_request_review".
+//
+// GitHub docs: https://developer.github.com/v3/activity/events/types/#pullrequestreviewevent
+type PullRequestReviewEvent struct {
+	// Action is always "submitted".
+	Action            *string            `json:"action,omitempty"`
+	PullRequestReview *PullRequestReview `json:"review,omitempty"`
```

Comment:
Suggestion. The JSON field is named `review`, it seems better to name the Go field the same to match:

```Go
Review *PullRequestReview `json:"review,omitempty"`
```